styles/style.css

```diff
@@ -0,0 +1,73 @@
+
+h1 {
+    font-family: Georgia, serif ;
+}
+
+h2 {
+    font-family: serif;
+}
+h3 {
```

Comment:
Since the properties here are the same as `h1`, they can be combined. It would look like this:
```
h1, h3 {
    font-family: Georgia, serif;
}
```